Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -128,6 +128,8 @@ After that, the integration will fetch incidents created after the timestamp of
 In case the **look-back** parameter is initialized with a certain value and during a time that incidents were fetched, if changing 
 the look back to a number that is greater than the previous value, then in the initial incident fetching there will be incidents duplications.
 If the integration was already set with look back > 0, and the look-back is not being increased at any point of time, then those incident duplications would not occur.
+Note that using a look back value that is very large (more than an hour) can lead to increase in the memory usage of the system.
```

Comment:
That is true **only** in case there are large number of service now tickets during that hour (I would say around 1000+).

however you are right we don't want to encourage customers to abuse this feature.
In case there aren't large number of tickets, then setting up large look-backs can be fine.